pkgs/development/libraries/spirv-headers/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "spirv-headers";
-  version = "1.5.3";
+  version = "1.5.4.raytracing.fixed";
 
   src = fetchFromGitHub {
     owner = "KhronosGroup";
     repo = "SPIRV-Headers";
     rev = version;
-    sha256 = "069sivqajp7z4p44lmrz23lvf237xpkjxd4lzrg27836pwqcz9bj";
+    hash = "sha256:WKDiLHSe5v2Ieh9Q1nad4lgvTHY6Ji5PkdJkxXAV94k=";
```

Comment:
```suggestion
    sha256 = "WKDiLHSe5v2Ieh9Q1nad4lgvTHY6Ji5PkdJkxXAV94k=";
```

Reply:
Is there some general convention around this? I have seen both variants used

Reply:
No, not really. Both should be fine but ``sha256 = "sha256:`` shouldn't be used.

Reply:
That's not what is here though?